packages/docusaurus-plugin-content-pages/src/index.ts

```diff
@@ -234,6 +235,15 @@ export default function pluginContentPages(
                         `${docuHash(aliasedSource)}.json`,
                       );
                     },
+                    // Assets allow to convert some relative images paths to
```

Comment:
Instead of a code comment here only, we might also consider a light mention about Assets and `require()` calls with `relative paths`, perhaps most especially in the API doc pages for `plugin-content-pages` and additionally maybe an Admonition on the `/docs/static-assets` page, which does mention `require()` but skips saying anything about `relative paths` ?  Anyways, just noting this since indeed `relative paths` are one of the tricky things that I had to figure out on my own with Docusaurus' default asset paths handling!  Maybe it's **Caveats** section could be expanded slightly, dunno.